MobiFlight/Config/MultiplexerDriver.cs

```diff
@@ -109,10 +103,9 @@ override public bool FromInternal(String value)
                 pins[1] = paramList[2];
                 pins[2] = paramList[3];
                 pins[3] = paramList[4];
-                res = Initialize(pins);
-            } else {
-                _initCounter++;
+                res = SetPins(pins);
```

Comment:
Why not setting the paramList result to the PinSx?

We don't have a logic that checks the pin values coming from the board in any other device either since we can assume that the configs are correct.

We an entirely remove SetPins(pins);

Reply:
> Why not setting the paramList result to the PinSx?

They are shifted by one; once shifted, we already have `SetPins()` available. If the issue is removing the validation check, see next comment;

> We don't have a logic that checks the pin values coming from the board in any other device either since we can assume that the configs are correct.

That's true - but that is also the reason why this bug could be caught in the first place. Maybe it's the other way around, checks could be implemented also in other places given the occasion.
 
> We an entirely remove SetPins(pins);

Do you mean the call or the method? Because it is used when initializing a new driver in the GUI.